docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
     environment:
      - BASE_URL=https://test-suite.pilot-prep.syncfor.science
     volumes:
-      - tests.db:/usr/src/app/testsuite/db
+      - tests.db:/db
       - ./config/tests/:/usr/src/app/config
```

Comment:
Yeah, http://docs.sqlalchemy.org/en/latest/core/engines.html#sqlite explained it to me (there are 4 slashes because  there's nominally an empty space path segment representing the hostname.